plugin-code/src/main/resources/template/css/cluecumber.css

```diff
@@ -21,11 +21,12 @@ html {
 
 
 .container {
-    max-width: 1080px;
+    max-width: 2000px;
```

Comment:
What is the reason for increasing the max-width? It was deliberately set to 1080.

Reply:
Why 1080, it is too small on big screen  and  it is why I increase it (not seems to be a problem, but I can remove it if you think it is not convenient with this PR)

Reply:
Fair point. Ok, you can leave it in there and I will test it further when your PR is merged. If it doesn't break anything we can leave it.

Reply:
Else I try and it break the feature index that is take into account to retreive the URL and I see multiple "Feature 2" but only ONE :( so I let the diff not taken into account